### PR TITLE
fix(NRQL): doc improvements to the NRQL querying section

### PR DESCRIPTION
**Pull request by jeff-colucci**

### CODE REVIEW

File: src/content/docs/nrql/nrql-references/nrql-math-using-select.mdx [hunk 1]
```diff
@@ -46,19 +46,15 @@ SELECT average(duration-databaseDuration) FROM Transaction
 
 ## Use advanced math operators with `SELECT` [#advancedmath]
 
-NRQL also includes some advanced mathematical capabilities that can be used for complex calculations. This is helpful if you want to process data to display it more effectively in the UI, or make statistical or psychometric calculations on queried results in a single step.
+NRQL also includes some advanced mathematical capabilities that can be used for complex calculations. This is helpful if you want to process data to display it more effectively in the UI, or make calculations on queried results in a single step.
```
Comment (paperclypse) [inline src/content/docs/nrql/nrql-references/nrql-math-using-select.mdx:49]: Given that we use capabilities to refer to our top-level features, maybe we want to use something else? Formulas, maybe?

File: src/content/docs/nrql/nrql-references/nrql-math-using-select.mdx [hunk 2]
```diff
@@ -46,19 +46,15 @@ SELECT average(duration-databaseDuration) FROM Transaction
...
 `abs(n)` returns the absolute value of _n_. For non-negative _n_ it returns _n_, and for negative _n_ it returns the positive number -_n_. For example `abs(2) = 2`, and `abs(-4) = 4`.
 
 ### `clamp_max`, `clamp_min` [#clamp]
 
-The clamping functions impose an upper or lower bound on values.
-
-For example, `clamp_max(duration, 10)` returns the duration, unless it exceeds 10, in which case 10 is returned.
-
-Similarly `clamp_min(duration, 1)` will not return any value lower than 1.
+The clamping functions impose an upper or lower bound on values. For example, `clamp_max(duration, 10)` returns the duration, unless it exceeds 10, in which case 10 is returned. Similarly `clamp_min(duration, 1)` will not return any value lower than 1.
```
Comment (paperclypse) [inline src/content/docs/nrql/nrql-references/nrql-math-using-select.mdx:57]: Marvelously clear and concise!

File: src/content/docs/nrql/nrql-references/nrql-math-using-select.mdx [hunk 3]
```diff
@@ -143,5 +137,5 @@ SELECT average(purchasePrice OR 0)
 This will replace NULL values with 0 or any number specified.
 
 <Callout variant="tip">
-  This can also be used to test whether something returns NULL or zero. `(zero) OR 1` returns 0. `(NULL) OR 1` returns 1.
+  You can also use this to whether something returns NULL or zero. `(zero) OR 1` returns 0. `(NULL) OR 1` returns 1.
```
Comment (paperclypse) [inline src/content/docs/nrql/nrql-references/nrql-math-using-select.mdx:140]: I feel like there's a missing word in "use this...to whether"

File: src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx [hunk 4]
```diff
@@ -6,29 +6,25 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-As part of New Relic's support for OpenTelemetry Protocol (OTLP) attributes can contain arrays. Arrays are a data structure that contains an ordered collection of values.
+As part of our support for OpenTelemetry Protocol (OTLP), attributes can contain arrays within New Relic. But what are arrays? Essentially, arrays are data structures containing an ordered collection of values that you can query with NRQL. Typically, arrays display in a format like `[1, 2, 3, 4, 5]` or `["US", "CA", "UK"]`.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx:9]: Should it be "the OpenTelemetry Protocol"?

Comment (paperclypse) [reply src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx:9]: There's something a little unclear about that first sentence. I'm assuming that OTLP can send data formatted as arrays to New Relic and New Relic can process and represent those arrays AS arrays, instead of munging them some other way. Is that what you're trying to convey? Perhaps there's a clearer way to state that.

Also, super torn about the array definition here, because it's such a basic/foundational programming concept

File: src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx [hunk 5]
```diff
@@ -6,29 +6,25 @@ redirects:
...
 
-Currently only simple arrays are supported. You can query them with NRQL. To quickly verify that any data is in an array, you can use the JSON chart type to see if the data displays as an array.
-
-## Simple arrays [#simple-arrays]
-
-Simple arrays are used to store a list of values of the same primitive type; for example, numeric, boolean, or string. An array looks like `[1, 2, 3, 4, 5]` or `["US", "CA", "UK"]`.
+Currently, we only support simple arrays. You can use simple arrays to store a list of values of the same primitive type; for example, numeric, boolean, or string. To quickly verify that an array contains any data, you can use the JSON chart type to see if the data displays in an array structure.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx:11]: Contra to my comment above, I think it's important to define what a simple arrays is vs more complex arrays.

Do you just mean that it's a single dimensional array and we don't support multi-dimensional ones?

File: src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx [hunk 6]
```diff
@@ -6,29 +6,25 @@ redirects:
...
 
 ## Supported features [#supported-features]
 
-* Array attributes are supported in the `FACET` clause.
-* The functions `getfield()`, `length()`, and `contains()` can be run on arrays.
-* The `uniques()`, `uniqueCount()`, and `latest()` aggregator functions are also supported.
+* The `FACET` clause supports array attributes.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx:15]: Do we need to clarify what array attributes we support?

File: src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx [hunk 7]
```diff
@@ -45,7 +41,7 @@ Simple arrays are used to store a list of values of the same primitive type; for
 
     `getfield(durations, 2)` will return `90`.
 
-    An alternative to the function `getfield()` is to use the square brackets `[ ]` shorthand.
+    Alternatively, you can use the square brackets `[ ]` shorthand instead of the function `getfield()`.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx:44]: I think you can just drop "alternatively" completely and other filler words like that

File: src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx [hunk 8]
```diff
@@ -65,7 +61,7 @@ Simple arrays are used to store a list of values of the same primitive type; for
     className="freq-link"
     title={<InlineCode>contains(attribute, element)</InlineCode>}
   >
-    Use the `contains()` function to determine if an element is present in an array.
+    Use the `contains()` function to determine the presence of an element in an array.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx:64]: I actually prefer the previous version slightly. 

"determine" is the word here that's bogging this sentence down. How about "see"? "Use the ... function to see if there's an element in your array."

File: src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx [hunk 9]
```diff
@@ -21,10 +21,10 @@ import queriesnrqlTimeseriesSpikyNRQLQueryBuilder from 'images/queries-nrql_scre
 
 import queriesnrqlTimeseriesSmoothedWithSlidebyNRQLQueryBuilder from 'images/queries-nrql_screenshot-full_timeseries-smoothed-with-slide-by-NRQL-query-builder.webp'
 
-Sliding windows are a technique for generating charts using the `SLIDE BY` clause in conjunction with the `TIMESERIES` clause. With sliding windows, data is gathered in time "windows" that overlap with each other.
+If you want to generate charts using the `SLIDE BY` clause in conjunction with the `TIMESERIES` clause, you can use a technique called sliding windows. This technique enables you to gather data in time "windows" that overlap with each other.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx:24]: I prefer "create" to generate

Comment (paperclypse) [reply src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx:24]: The definition feels a bit circular. Is there anyway you can define it a different way without using "windows"?

File: src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx [hunk 10]
```diff
@@ -39,7 +39,7 @@ For example, in the image below, a query gathers data with 5 minute windows. The
   5-minute windows with 1-minute "slide"
 </figcaption>
 
-In contrast, with "tumbling" or "cascading" windows, the windows do not overlap. For example, in this `TIMESERIES 3 minutes` NRQL query, the windows are 3 minutes in length, with each beginning when the other ends. There is no overlap in the measurement interval.
+In contrast to sliding windows, "tumbling" or "cascading" windows don't overlap. For example, in this `TIMESERIES 3 minutes` NRQL query, the windows are 3 minutes in length, with each beginning when the other ends without overlap in the measurement interval.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx:42]: "without an overlap"?

File: src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx [hunk 11]
```diff
@@ -53,7 +53,7 @@ In contrast, with "tumbling" or "cascading" windows, the windows do not overlap.
 
 ## When to use sliding windows [#when-to-use]
 
-Sliding windows are helpful when you need to smooth out "spiky" charts. One common use case is to use sliding windows to smooth line graphs that have a lot of variation over short periods of time in cases where the rolling aggregate (for example a rolling mean) is more important than aggregates from narrow windows of time.
+Sliding windows are helpful when you need to smooth out "spiky" charts. It's common to use sliding windows to smooth line graphs that have a lot of variation over short periods of time in cases where the rolling aggregate (for example a rolling mean) is more important than aggregates from narrow windows of time.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx:56]: "It's common..." --> "You can use sliding windows to smooth spiky line graphs that have many variations over short periods of time. Especially use this when the rolling aggregate of the data is more important to you than etcetc"

File: src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx [hunk 12]
```diff
@@ -79,27 +79,29 @@ However, in this example, 5-minute wide `TIMESERIES` windows are smoothed with t
   TIMESERIES query with SLIDE BY clause
 </figcaption>
 
-## Valid NRQL syntax for `SLIDE BY` [#valid-syntax]
+## `SLIDE BY` syntax [#valid-syntax]
 
-Valid NRQL syntax for the `SLIDE BY` clause will follow the format below.
+Valid NRQL syntax for the `SLIDE BY` clause follows the format below.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx:84]: Much better :) see how dropping the "will" makes the sentence feel more direct and concise?

File: src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx [hunk 13]
```diff
@@ -79,27 +79,29 @@ However, in this example, 5-minute wide `TIMESERIES` windows are smoothed with t
...
-## Translation from PromQL-style queries [#translate-promql]
+<Callout variant="tip">
+  When paired with `SLIDE BY`, `TIMESERIES` does not support `AUTO` or `MAX`. The `TIMESERIES` value must be an integer time unit value. In other words, `SLIDE BY AUTO` or `SLIDE BY MAX` will work, but `TIMESERIES AUTO` or `TIMESERIES MAX` followed by `SLIDE BY` and `MAX`, `AUTO`, or a specific integer time unit is not supported.
+</Callout>
 
-When applicable, a PromQL-style query is translated into a NRQL sliding window query.
+## Translation from PromQL-style queries [#translate-promql]
 
-For example, if your PromQL style query uses `rate(request_count[5m])` for the past 60 minutes with a 1-minute window overlap, the NRQL translation would be the query below.
+When applicable, a PromQL-style query is translated into a NRQL sliding window query. For example, if your PromQL style query uses `rate(request_count[5m])` for the past 60 minutes with a 1-minute window overlap, the NRQL translation would be the query below.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx:104]: "would be the query below" feels a bit cumbersome. How about: "...1-minute overlap. Here's that PromQL translated into NRQL:"

File: src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx [hunk 14]
```diff
@@ -18,11 +18,9 @@ freshnessValidatedDate: never
 
 import queriesnrqlFunnelNRQL from 'images/queries-nrql_screenshot-crop_funnel-NRQL.webp'
 
-With [NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql), you can use funnels to evaluate sets of related actions. The actions generally share a relationship via an identifier such as a user ID or session ID.
+With [NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql), you can use funnels to evaluate sets of related actions. The actions generally share a relationship with an identifier, such as a user ID or session ID.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx:21]: I think replacing "via" actually muddies the waters here a bit. I do approve of the comma tho

File: src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx [hunk 15]
```diff
@@ -40,23 +38,21 @@ FROM DATA_TYPE
   SINCE TIMEFRAME
 ```
 
-See also a more complex [funnel query example](#example).
-
 Funnel queries require the [`funnel` function](/docs/query-data/nrql-new-relic-query-language/getting-started/nrql-syntax-components-functions#func-funnel), an [attribute](/docs/using-new-relic/welcome-new-relic/get-started/glossary#attribute) to funnel, and at least two steps:
 
-1. The first step is the anchor step, which will always represent 100% of the results.
-2. The second and later steps describe the number of users who have also completed additional actions. This number typically will be less than 100%. However, it could be 100% if every user who completes action (A) also completes the additional actions being queried.
+1. The anchor step, which always represents 100% of the results.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx:43]: Is there a way to turn this into a complete sentence?

Comment (paperclypse) [reply src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx:43]: And in 2 as well

File: src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx [hunk 16]
```diff
@@ -40,23 +38,21 @@ FROM DATA_TYPE
...
+2. One or more steps that describe the number of users who have also completed additional actions, typically than the 100% from the anchor step. However, it could be 100% if every user who completes action (A) also completes the additional actions you're querying.
 
 ## Funnel query technical details
 
-One way to use funnel queries is to calculate the rate at which multi-step, ordered sequences were completed over a given timeframe, like in the New Relic signup [example](#why-matters). But you can also use funnels to calculate overlap between several distinct actions that don't have a sequential relationship to one another. In other words, the order of steps doesn't impact the calculations performed.
+You can use funnel queries to calculate the completion rate for which multi-step, ordered sequences over a given timeframe, like in the New Relic signup [example](#why-matters). You can also use funnels to calculate overlap between several distinct actions that don't have a sequential relationship to one another. In other words, the order of steps won't impact the calculations performed.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx:48]: I can't quite make sense of that first sentence there.

Comment (paperclypse) [reply src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx:48]: also, the end of the paragraph is too passive. How about simply: "the order of steps won't impact the calculations." (the performed being implied)

File: src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx [hunk 17]
```diff
@@ -40,23 +38,21 @@ FROM DATA_TYPE
...
 
 The way funnel queries compute results varies depending on the size of the data set:
 
-* If the funnel is dealing with fewer than 256 funnel attribute values, it will calculate the value exactly.
-* If it's dealing with 256 or more funnel attribute values, it applies an algorithm called MinHash to calculate approximate results for optimized performance.
+* If the funnel deals with fewer than 256 funnel attribute values, it will calculate the value exactly.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx:52]: "deals" is a funny word here. maybe "has" is better? or "is processing"?

File: src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx [hunk 18]
```diff
@@ -40,23 +38,21 @@ FROM DATA_TYPE
...
+* If the funnel deals with 256 or more funnel attribute values, it applies an algorithm called MinHash to calculate approximate results for optimized performance.
 
-This means that for large data sets, there may be occasional "false positives" at the level of individual data points. These do not interfere greatly with the accuracy of the numerical estimates provided in query results.
+This means that for large data sets, there may be occasional "false positives" at the level of individual data points. These don't interfere greatly with the accuracy of the numerical estimates provided in query results.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx:55]: I'd be a bit troubled by the hand-waving "interfere greatly" here if I were reading this, I think

File: src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx [hunk 19]
```diff
@@ -116,9 +112,9 @@ Here's a detailed breakdown of technical details and constraints for funnel quer
   </tbody>
 </table>
 
-## Funnel query example [#example]
+## Funnel example [#example]
 
-This example queries the <InlinePopover type="browser" /> [`PageView`](/attribute-dictionary/?event=PageView) event and its attributes. It queries unique browser sessions that have progressed from browsing a product, to adding to cart, to checkout. Labels are included for each step, indicated by the keyword `AS`.
+This example queries the <InlinePopover type="browser" /> [`PageView`](/attribute-dictionary/?event=PageView) event and its attributes. It queries unique browser sessions that have progressed from browsing for products, to adding them to their cart, to checking out. It includes labels for each step, indicated by the keyword `AS`.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx:117]: Is there a reason humanize this? It's people who are doing these things.

"It queries unique browser sessions from your users as they progress through browsing for products, etc etc"

File: src/content/docs/nrql/using-nrql/lookups.mdx [hunk 20]
```diff
@@ -16,31 +16,45 @@ import nrqlLookupQueryTranslateItemIds from 'images/nrql_screenshot-crop_lookup-
 
 import nrqlLookupQueryGeoip from 'images/nrql_screenshot-crop_lookup-query-geoip.webp'
 
-When you [upload CSV-format lookup tables](/docs/logs/ui-data/lookup-tables-ui), you can use the `lookup()` function to access that data in your NRQL queries.
-
-## Why use lookup tables? [#why]
-
-For why you'd use lookup tables and how to upload them, see the [Lookup tables overview](/docs/logs/ui-data/lookup-tables-ui/#overview).
+When you [upload CSV-format lookup tables](/docs/logs/ui-data/lookup-tables-ui), you can use the `lookup()` function to access that data in your NRQL queries. You can use lookup tables to query data that's not in your New Relic account, help you parse telemetry data, and customize your data groupings.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/lookups.mdx:19]: Once you've uploaded a CSV to NR isn't it then in your NR account? There's something odd worth clarifying there

File: src/content/docs/nrql/using-nrql/lookups.mdx [hunk 21]
```diff
@@ -16,31 +16,45 @@ import nrqlLookupQueryTranslateItemIds from 'images/nrql_screenshot-crop_lookup-
...
 
 ## Basic query syntax [#basic-syntax]
 
-Let's say you've named your table `storeNames`. This query will select all data from that table:
+The following lookup query has the basic syntax for this function. It uses a table named `storeNames` and selects all data from that table:
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/lookups.mdx:23]: Or even just "This lookup query shows the basic syntax for the lookup() function."

Comment (paperclypse) [reply src/content/docs/nrql/using-nrql/lookups.mdx:23]: It's ok to repeat proper nouns if it makes things more clear :)

File: src/content/docs/nrql/using-nrql/lookups.mdx [hunk 22]
```diff
@@ -16,31 +16,45 @@ import nrqlLookupQueryTranslateItemIds from 'images/nrql_screenshot-crop_lookup-
...
 
 ```sql
 FROM lookup(storeNames)
 SELECT *
 ```
 
-This query will select some specific attributes from that same table:
+This query selects some specific attributes from that same table:
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/lookups.mdx:30]: drop the "some" I think

File: src/content/docs/nrql/using-nrql/lookups.mdx [hunk 23]
```diff
@@ -16,31 +16,45 @@ import nrqlLookupQueryTranslateItemIds from 'images/nrql_screenshot-crop_lookup-
...
+
+Lookup tables support a higher [`LIMIT`](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/#sel-limit) when querying than other NRQL data types do. You can set a `LIMIT` of up to 20,000 when using a lookup table within a query, [subquery](/docs/query-your-data/nrql-new-relic-query-language/get-started/subqueries-in-nrql/), and [nested aggregation](/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/nested-aggregation-make-ordered-computations-single-query/).
+
+<Callout variant="tip">
+When you use a lookup table within a subquery, the outer query `LIMIT` is bound by the standard maximum value unless it also uses a lookup table.
+</Callout>
+
+## Technical limitations [#limitations]
+
+* You can't use lookup data for NRQL alert conditions.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/lookups.mdx:47]: "for" -> "with"?

File: src/content/docs/nrql/using-nrql/lookups.mdx [hunk 24]
```diff
@@ -64,15 +78,15 @@ WHERE hostname IN (FROM lookup(myHosts) SELECT uniques(myHost))
...
+Using `JOIN` queries can make your data easier to understand. For example, this query for a custom event type uses the `storeNames` table to show the store names along with the total sales.
 
 ```sql
 FROM StoreEvent
 JOIN (FROM lookup(storeNames) SELECT store_ID as storeId, storeName AS name LIMIT 10000) ON shopId = storeId
 SELECT shopId, storeName, totalSales
 ```
 
-Also notice this allows a limit of 10,000: that's because lookup tables support a higher limit than other NRQL data types. For more details, see the [query limits](#query-limits) section below.
+Notice that this allows a limit of 10,000: that's because lookup tables support a higher limit than other NRQL data types, as is mentioned in the [query limits](#query-limits) section.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/lookups.mdx:89]: I think you can drop "notice that" and go right into "this"

Comment (paperclypse) [reply src/content/docs/nrql/using-nrql/lookups.mdx:89]: Also, the : is odd there. I think it's fine to break it up into two sentences.

File: src/content/docs/nrql/using-nrql/query-infrastructure-dimensional-metrics-nrql.mdx [hunk 25]
```diff
@@ -16,25 +16,17 @@ import queriesnrqlDimensionalMetricsNamingConvention from 'images/queries-nrql_d
...
-
-At New Relic we report [metrics](/docs/using-new-relic/data/understand-data/new-relic-data-types#metrics) in several ways, including [dimensional metrics](/docs/using-new-relic/data/understand-data/new-relic-data-types#dimensional-metrics), which are used by our metric API, Telemetry SDK, some open-source integrations, and our infrastructure services.
+At New Relic, we report [metrics](/docs/using-new-relic/data/understand-data/new-relic-data-types#metrics) in several ways, including [dimensional metrics](/docs/using-new-relic/data/understand-data/new-relic-data-types#dimensional-metrics), which our metric API, Telemetry SDK, some open-source integrations, and our infrastructure services can all use.
 
 This type of metric enables you to:
 
 * Enjoy an improved query experience for infrastructure data.
 * Discover all your metrics in one place.
-* Tap into more metric sources, such as [Prometheus](/docs/integrations/prometheus-integrations/get-started/new-relic-prometheus-openmetrics-integration-kubernetes).
+* Utilize more metric sources, such as [Prometheus](/docs/integrations/prometheus-integrations/get-started/new-relic-prometheus-openmetrics-integration-kubernetes).
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/query-infrastructure-dimensional-metrics-nrql.mdx:27]: "utilize" --> "use"

File: src/content/docs/nrql/using-nrql/query-time-range.mdx [hunk 26]
```diff
@@ -61,7 +58,7 @@ You have a few ways to specify time in the `SINCE` and until `UNTIL` clauses.
 </Callout>
 
 ### Unix time (Epoch Milliseconds) [#time-n-units-ago]
-The time can be referenced as Unix time in milliseconds. This is the number of milliseconds since `1970-01-01T00:00:00Z`, or midnight on the 1st of January, 1970. For example:
+You can have the time reference as Unix time in milliseconds. This is the number of milliseconds since `1970-01-01T00:00:00Z`, or midnight on the 1st of January, 1970. For example:
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/query-time-range.mdx:61]: That first sentence is oddly structured. I needed the previous version to understand it.

File: src/content/docs/nrql/using-nrql/query-time-range.mdx [hunk 27]
```diff
@@ -126,7 +123,7 @@ The time is relative to the time you run the query. For example, assume that you
   </table>
 
 ### Relative temporal time [#relative-temporal-time]
-Relative time can also be specified using temporal time references. These relative time references resolve to the beginning of the temporal time interval. A relative temporal time reference will honor a `WITH TIMEZONE` clause. Select a collapser below to see more information on specific temporal time references:
+You can also specify relative time using temporal time references. These relative time references resolve to the beginning of the temporal time interval. A relative temporal time reference will honor a `WITH TIMEZONE` clause. Select a collapser below to see more information on specific temporal time references:
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/query-time-range.mdx:126]: "specify" --> "use"?

File: src/content/docs/nrql/using-nrql/subqueries-in-nrql.mdx [hunk 28]
```diff
@@ -27,30 +21,30 @@ SELECT count(*) FROM Transaction WHERE duration > (SELECT average(duration) FROM
...
 
 ## Subquery execution [#subquery-execution]
 
-During query execution, each subquery is run independently, and its result is used as a constant value, or set of values, in the outer query. Due to this execution model, subqueries may not reference attributes and values from the outer query.
+During query execution, each subquery runs independently, and its result use as a constant value, or set of values, in the outer query. Due to this execution model, subqueries may not reference attributes and values from the outer query.
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/subqueries-in-nrql.mdx:30]: "its result use as a constant value" should it be "is used"?

File: src/content/docs/nrql/using-nrql/subqueries-in-nrql.mdx [hunk 29]
```diff
@@ -59,9 +53,9 @@ Here are some example subqueries:
     id="numeric-conditions"
     title="Numeric conditions"
   >
-    Any query which returns a single numeric value, such as those using the aggregators `average()`, `count()`, `uniqueCount()`, or `latest()`, can be used in numeric conditions.
+    You can use any query which returns a single numeric value, such as those using the aggregators `average()`, `count()`, `uniqueCount()`, or `latest()` in numeric conditions.
 
-    Find the count of transactions whose duration was greater than average:
+    The query below finds the count of transactions whose duration was greater than average:
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/subqueries-in-nrql.mdx:58]: I don't think you need this "The query below finds" phrasing. I slightly prefer the previous wording

File: src/content/docs/nrql/using-nrql/subquery-joins.mdx [hunk 30]
```diff
@@ -22,23 +22,19 @@ import nrqlJoinTuturialCalculationsSidebyside from 'images/nrql-join-tutorial-ca
...
+This example joins infrastructure data with transaction data:
 
 <img
   title="Datasets related by container ID"
   alt="Datasets Related by Container ID"
   src={nrqlJoinTuturialRowsSidebyside}
 />
 
-In this example there are two datasets: the average CPU percentage of infrastructure containers (`ProcessSample`) and the average duration of app transactions by container. Often, data from different sources is correlated. In this case you can determine if a container's higher CPU usage is causing slower transactions by using the following subquery join:
+In this example above contains two datasets: the average CPU percentage of infrastructure containers (`ProcessSample`) and the average duration of app transactions by container. Often, data from different sources is correlated. In this case, you can determine if a container's higher CPU usage is causing slower transactions by using the following subquery join:
```
Comment (paperclypse) [inline src/content/docs/nrql/using-nrql/subquery-joins.mdx:37]: Is the example really above? Do you mean the image above?

**Review by paperclypse — APPROVED** [review]
Copyedit pass done. Flagged some minor things.

**Comment by jeff-colucci** [comment]
Merging with approval for skipping re-review
